vue-components/src/components/LookupMenu.vue

```diff
@@ -47,8 +50,26 @@ export default Vue.extend( {
 				rootElem.style.maxHeight = menuHeight + 'px';
 			}
 		},
-	},
+		onScroll: debounce( function ( this: Vue ) {
+			const rootElem = this.$el as HTMLElement;
+			const menuItems = rootElem.querySelectorAll( '.wikit-LookupMenu__item' ) as NodeListOf<HTMLElement>;
```

Comment:
The "Vue way" of doing this is to use [ref](https://vuejs.org/v2/api/#ref).
An added bonus of using `ref` is future-proofing this component for the requirement to be server side rendered.

Reply:
Great catch, we will need some refactoring in other components then 😅